CTP/README.md

```diff
@@ -58,16 +75,15 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 	    ```
 	    $ bin/ctp.sh medium -c ./conf/medium.conf
 	    ```

```

Comment:
-> please separate the example. like below:
	* For **SQL** test:
	    ```
	    $ bin/ctp.sh sql -c ./conf/sql.conf
	    ```
	* For **Medium** test:
	    ```
	    $ bin/ctp.sh medium -c ./conf/medium.conf
            ```